extractors/README.md

```diff
@@ -0,0 +1,25 @@
+# Extractors
+
+The scripts in this folder will extract the data from the best datasource we can find in order to have a realist run of
+the simple network sim model. They are not necessarily downloading them from the official SCRCdata resource, although
+they might be partially doing that. The resulting files from the scripts in here can be plugged into the model by
+coppying them to the appropriate folder inside `sample_input_files`.
```

Comment:
coppying->copying

Reply:
presumably also realist-> realistic, unless our concerns have shifted to the artistic

Reply:
any move away from the surrealist here is to be encouraged